twake/backend/node/src/services/messages/services/messages-operations.ts

```diff
@@ -9,15 +9,19 @@ import {
   ReactionOperation,
   ThreadExecutionContext,
 } from "../types";
-
+import emoji from "emoji-name-map";
 import Repository from "../../../core/platform/services/database/services/orm/repository/repository";
 import { ThreadMessagesService } from "./messages";
 import gr from "../../global-resolver";
 import { updateMessageReactions } from "../../../utils/messages";
 import { localEventBus } from "../../../core/platform/framework/event-bus";
-
+import { PushReactionNotification } from "../../../services/notifications/services/engine/processors/reaction-notification";
+import { ReactionNotification } from "../../../services/notifications/types";
 export class ThreadMessagesOperationsService {
-  constructor(private threadMessagesService: ThreadMessagesService) {}
+  private reactionNotifications: PushReactionNotification;
+  constructor(private threadMessagesService: ThreadMessagesService) {
+    this.reactionNotifications = new PushReactionNotification();
```

Comment:
This should go (as well as the corresponding import)

Reply:
done